### PR TITLE
Convert pyramids to Raycast sensor

### PR DESCRIPTION
**Pull request by chriselion**
Removing the last usage of RayPerception (the non-sensor-based approach) from the examples and retrain the network.

After this and https://github.com/Unity-Technologies/ml-agents/pull/3297 we can remove the deprecated code.

### CODE REVIEW

File: com.unity.ml-agents/csharp_timers.json.meta [hunk 1]
```diff
@@ -1,7 +0,0 @@
-fileFormatVersion: 2
```
Comment (chriselion) [inline com.unity.ml-agents/csharp_timers.json.meta]: This probably shouldn't have gotten added, might need to add it to .gitignore

File: Project/Assets/ML-Agents/Examples/Pyramids/Scripts/PyramidAgent.cs [hunk 2]
```diff
@@ -19,23 +18,13 @@ public override void InitializeAgent()
...
-        m_RayPer = GetComponent<RayPerception>();
         m_SwitchLogic = areaSwitch.GetComponent<PyramidSwitch>();
     }
 
     public override void CollectObservations()
     {
         if (useVectorObs)
         {
-            const float rayDistance = 35f;
-            float[] rayAngles = { 20f, 90f, 160f, 45f, 135f, 70f, 110f };
```
Comment (chriselion) [inline Project/Assets/ML-Agents/Examples/Pyramids/Scripts/PyramidAgent.cs]: I matched the maxRayDegrees in the new component to the min/max of the old ones, but the exact distribution won't be the same.

Comment (surfnerd) [reply Project/Assets/ML-Agents/Examples/Pyramids/Scripts/PyramidAgent.cs]: does it need to be retrained?

Comment (chriselion) [reply Project/Assets/ML-Agents/Examples/Pyramids/Scripts/PyramidAgent.cs]: Yes, new Pyramids.nn file is in PR. I ran it last night.

**Pull request merged**